Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/API/UI/LeaderBoard.hs

```diff
@@ -10,15 +10,23 @@ import Servant
 import Tools.Auth
 
 type API =
-  "driver"
-    :> "leaderBoard"
-    :> TokenAuth
-    :> QueryParam "limit" Integer
-    :> Get '[JSON] DLeaderBoard.LeaderBoardRes
+  "driver" :> "leaderBoard"
+    :> ( TokenAuth
+           :> QueryParam "limit" Integer
+           :> Get '[JSON] DLeaderBoard.LeaderBoardRes
+           :<|> TokenAuth
+           :> QueryParam "limit" Int
+           :> ReqBody '[JSON] DLeaderBoard.LeaderBoardReq
```

Comment:
This can be a QueryParam and the request can be GET

Reply:
Getting date for leaderBoard as requestBody

Reply:
Why? Can we not make it a query param?

Reply:
In the same requestBody I was thinking of adding dates for weekly leaderBoard, which will have **from** and **to** and if we have only **date** we will only seek the daily leaderBoard